pkg/netflow/common/utils.go

```diff
@@ -23,6 +23,14 @@ func MaxUint64(a uint64, b uint64) uint64 {
 	return b
 }
 
+// MaxUint16 returns the max of the two passed number
+func MaxUint16(a uint16, b uint16) uint16 {
```

Comment:
> Why not using math.Max ? Because of conversion to float ?

Yes, that's right. 

> This sounds like a function other teams could use, do we have any common math utils ? Or math package extender ?

I'm not seeing any common math utils in `datadog-agent`. There are external libraries, but I'm not sure it's worth importing them simple function like this one.

With Go, 1.18, we can possibly use generics instead: https://stackoverflow.com/a/27516559